testfile.txt

```diff
@@ -0,0 +1 @@
+text in a test file
```

Comment:
what's this?

Reply:
I have no idea.  I'll have to clean that up; it is probably something that should have been in the gitignore.

Reply:
This shouldn't be here any more.